06_PDDE_Checker.R

```diff
@@ -437,23 +469,115 @@ overlapping_hmis_participation <- HMISParticipation %>%
   merge_check_info(checkIDs = 131) %>%
   select(all_of(PDDEcols))
 
+
+# Bed Type incompatible with Housing Type -----------------------------------
+#If HousingType is 1 or 2 (site-based), then BedType should be 1 (facility based beds). If HousingType is 3 (tenant-based), then BedType should be 2 (voucher beds).
+
+ES_BedType_HousingType <- Inventory %>%
+  left_join(Project0(), by = "ProjectID") %>%
+  left_join(HousingTypeDF, by = "ProjectID") %>% 
+  filter(ProjectType %in% c(0, 1) &
```

Comment:
Not sure why you're filtering to ES projects, but if that's intentional then you should:
1. Note it in the comments.
2. Replace the 0 with the hardcodes.R variable, `es_ee_project_type`, and replace the 1 with `es_nbn_project_type`. This makes it more clear which projects you're selecting.

Reply:
done